### ISSUE TITLE
No error messages raised when dataset name is not found

### ISSUE THREAD

**Issue by BrendenBarbour** [mit-quest/necstlab-damage-segmentation#87]
Causes various errors when a dataset that does not exist is called in multiple locations, train_segmentation_model, etc.

**Comment by CarolinaFurtado** [comment]
what to take this one, @BrendenBarbour? I think doing something like: 

```
try:
    copy_folder_locally_if_missing(os.path.join(gcp_bucket, 'datasets', train_config['dataset_id']),
                                   local_dataset_dir)
except ...
     some type of error
```
in train_segmentation_models.py does the trick. Maybe we could take this opportunity to do the other steps of the workflow. 

If you are busy with other things, I can tackle these.

**Comment by CarolinaFurtado** [comment]
This will be addressed in issue #91
